ui/desktop/app/controllers/application.js

```diff
@@ -15,14 +17,34 @@ export default class ApplicationController extends Controller {
   @service clusterUrl;
   @service flashMessages;
 
+  // =tracked properties
+
+  @tracked showSignoutModal = false;
+  @tracked showCloseSessionsModal = false;
```

Comment:
It might make more sense to call these something like `isLoggingOut` and `isAppQuitting` if we can use one modal